x-pack/plugins/secops/public/components/timeline/ecs.ts

```diff
@@ -8,37 +8,37 @@ export interface ECS {
   _id: string;
   timestamp: string;
   host: {
-    hostname: string;
-    ip: string;
+    hostname?: string;
+    ip?: string;
   };
   event: {
-    id: string;
-    category: string;
-    type: string;
-    module: string;
-    severity: number;
+    id?: string;
+    category?: string;
+    type?: string;
+    module?: string;
+    severity?: number;
   };
   suricata?: {
-    eve: {
-      flow_id: number;
-      proto: string;
-      alert: {
+    eve?: {
+      flow_id?: number;
+      proto?: string;
+      alert?: {
         signature: string;
         signature_id: number;
       };
     };
   };
   source: {
-    ip: string;
-    port: number;
+    ip?: string;
+    port?: number;
   };
   destination: {
-    ip: string;
-    port: number;
+    ip?: string;
+    port?: number;
   };
   geo: {
-    region_name: string;
-    country_iso_code: string;
+    region_name?: string;
+    country_iso_code?: string;
```

Comment:
Was on a zoom call during this part. Yea, this is needed as the data puts `null` in all of these places, so this is all needed at the moment unless we do an adapter at the network endpoint with Apollo and I don't think we want to do that.

Luckily the renderers do their job on how to handle all of this logic.